tests/lit-tests/2280.ispc

```diff
@@ -0,0 +1,30 @@
+// RUN:     %{ispc} --target=host --nowrap --nostdlib -g -O0 -o %t.o %s -E
```

Comment:
I suggest checking specific things that where done right and can be detected in the preprocessor out put, specifically:
- inlining done right
- macros are substituted. I.e. pass `-DMYMACRO=MACRO_VALUE` and detect that `MACRO_VALUE` presents in the output.